helix-core/src/syntax.rs

```diff
@@ -63,6 +64,8 @@ pub struct LanguageConfiguration {
 
     #[serde(default)]
     pub auto_format: bool,
+    #[serde(default)]
+    pub diagnostic_severity: Severity,
```

Comment:
This also needs a description in `book/configuration.md`

Reply:
Added.